docs/data-types/lists.md

```diff
@@ -35,60 +35,60 @@ See the [complete series of list commands](https://redis.io/commands/?group=list
 ## Examples
 
 * Treat a list like a queue (first in, first out):
```

Comment:
We can make this example more self-explanatory by splitting it into two steps:
1. Producer:  LPUSH, RPUSH, etc.
2. Consumer: LPOP, RPOP, LRANGE, etc.